backend/src/main/java/hu/flowacademy/konyhatunder/controller/RecipeController.java

```diff
@@ -28,8 +31,8 @@ public Recipe getRecipe(@PathVariable String id) {
 
     @PostMapping
     @ResponseStatus(HttpStatus.CREATED)
-    public Recipe createRecipe(@RequestBody RecipeDTO recipeDTO) {
-        return recipeService.createRecipe(recipeDTO);
+    public Recipe createRecipe(@RequestPart String recipeDTO, @RequestPart(name = "image", required = false) MultipartFile image) throws JsonProcessingException {
```

Comment:
Majd itt is jobb lenne ha ki lenne mappelve a dto, es teljesen a jacksonra lenne bizva a parsolas, es nem stringgel buveszkednetek, de ez is mehet masik issue-ba.